packages/extension-ui/src/Popup/AuthManagement/index.tsx

```diff
@@ -144,6 +144,10 @@ export default styled(AuthManagement)`
   display: flex;
   flex-direction: column;
 
+  &&& {
+    padding-inline: 0 !important;
```

Comment:
This is where I give up - been jugling styles breaking each other back and forth entire day and trying to make it work properly other than overhauling the entire views system of this project is a waste of time.